### ISSUE TITLE
WASM: fatal error: stack overflow

### ISSUE THREAD

**Issue by 88250** [tinygo-org/tinygo#527]
node.go: 

```go
package main

type Node interface {
	Parent() Node
}

type BaseNode struct {
	parent          Node
}

func (n *BaseNode) Parent() Node {
	return n.parent
}

func (n *BaseNode) SetParent(parent Node) {
	n.parent = parent
}
```

main.go:

```go
package main

import (
	"fmt"
)

//go:export md
func md(markdownText string) string {
	fmt.Println("TinyGo！")
	var node Node // use interface Node not work
	//var node *BaseNode // use struct *BaseNode works
	node = &BaseNode{}
	_ = node
	return "TinyGo！"
}

func main() {
	select {}
}
```

build script: 

```
tinygo build -o lute.tinygo.wasm -target wasm
```

error log:

```
runtime: goroutine stack exceeds 1000000000-byte limit
fatal error: stack overflow

runtime stack:
runtime.throw(0x34a8749, 0xe)
        /usr/local/go/src/runtime/panic.go:617 +0x72
runtime.newstack()
        /usr/local/go/src/runtime/stack.go:1041 +0x6f0
runtime.morestack()
        /usr/local/go/src/runtime/asm_amd64.s:429 +0x8f

goroutine 1 [running]:
runtime.newobject(0x345bc00, 0x0)
        /usr/local/go/src/runtime/malloc.go:1067 +0x51 fp=0xc024000338 sp=0xc024000330 pc=0x8acf81
go/types.TypeString(0x35251e0, 0xc0015db7d0, 0x0, 0x0, 0xc0004ce000)
        /usr/local/go/src/go/types/typestring.go:66 +0x3f fp=0xc024000408 sp=0xc024000338 pc=0xa4f2bf
go/types.(*Named).String(...)
        /usr/local/go/src/go/types/type.go:490
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958c60)
        /home/circleci/project/compiler/interface.go:63 +0x1010 fp=0xc0240005b8 sp=0xc024000408 pc=0xbabda0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958c50)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024000768 sp=0xc0240005b8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958c40)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024000918 sp=0xc024000768 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958c30)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024000ac8 sp=0xc024000918 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958c20)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024000c78 sp=0xc024000ac8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958c10)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024000e28 sp=0xc024000c78 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958c00)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024000fd8 sp=0xc024000e28 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958bf0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024001188 sp=0xc024000fd8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958be0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024001338 sp=0xc024001188 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958bd0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240014e8 sp=0xc024001338 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958bc0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024001698 sp=0xc0240014e8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958bb0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024001848 sp=0xc024001698 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958ba0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240019f8 sp=0xc024001848 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958b90)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024001ba8 sp=0xc0240019f8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958b80)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024001d58 sp=0xc024001ba8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958b70)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024001f08 sp=0xc024001d58 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958b60)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240020b8 sp=0xc024001f08 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958b50)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024002268 sp=0xc0240020b8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958b40)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024002418 sp=0xc024002268 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958b30)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240025c8 sp=0xc024002418 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958b20)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024002778 sp=0xc0240025c8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958b10)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024002928 sp=0xc024002778 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958b00)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024002ad8 sp=0xc024002928 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958af0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024002c88 sp=0xc024002ad8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958ae0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024002e38 sp=0xc024002c88 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958ad0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024002fe8 sp=0xc024002e38 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958ac0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024003198 sp=0xc024002fe8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958ab0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024003348 sp=0xc024003198 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958aa0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240034f8 sp=0xc024003348 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958a90)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240036a8 sp=0xc0240034f8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958a80)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024003858 sp=0xc0240036a8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958a70)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024003a08 sp=0xc024003858 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958a60)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024003bb8 sp=0xc024003a08 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958a50)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024003d68 sp=0xc024003bb8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958a40)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024003f18 sp=0xc024003d68 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958a30)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240040c8 sp=0xc024003f18 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958a20)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024004278 sp=0xc0240040c8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958a10)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024004428 sp=0xc024004278 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958a00)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240045d8 sp=0xc024004428 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029589f0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024004788 sp=0xc0240045d8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029589e0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024004938 sp=0xc024004788 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029589d0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024004ae8 sp=0xc024004938 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029589c0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024004c98 sp=0xc024004ae8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029589b0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024004e48 sp=0xc024004c98 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029589a0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024004ff8 sp=0xc024004e48 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958990)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240051a8 sp=0xc024004ff8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958980)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024005358 sp=0xc0240051a8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958970)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024005508 sp=0xc024005358 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958960)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240056b8 sp=0xc024005508 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958950)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024005868 sp=0xc0240056b8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958940)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024005a18 sp=0xc024005868 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958930)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024005bc8 sp=0xc024005a18 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958920)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024005d78 sp=0xc024005bc8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958910)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024005f28 sp=0xc024005d78 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958900)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240060d8 sp=0xc024005f28 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029588f0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024006288 sp=0xc0240060d8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029588e0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024006438 sp=0xc024006288 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029588d0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240065e8 sp=0xc024006438 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029588c0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024006798 sp=0xc0240065e8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029588b0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024006948 sp=0xc024006798 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029588a0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024006af8 sp=0xc024006948 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958890)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024006ca8 sp=0xc024006af8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958880)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024006e58 sp=0xc024006ca8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958870)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024007008 sp=0xc024006e58 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958860)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240071b8 sp=0xc024007008 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958850)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024007368 sp=0xc0240071b8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958840)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024007518 sp=0xc024007368 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958830)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240076c8 sp=0xc024007518 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958820)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024007878 sp=0xc0240076c8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958810)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024007a28 sp=0xc024007878 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958800)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024007bd8 sp=0xc024007a28 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029587f0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024007d88 sp=0xc024007bd8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029587e0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024007f38 sp=0xc024007d88 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029587d0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240080e8 sp=0xc024007f38 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029587c0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024008298 sp=0xc0240080e8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029587b0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024008448 sp=0xc024008298 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029587a0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240085f8 sp=0xc024008448 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958790)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240087a8 sp=0xc0240085f8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958780)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024008958 sp=0xc0240087a8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958770)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024008b08 sp=0xc024008958 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958760)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024008cb8 sp=0xc024008b08 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958750)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024008e68 sp=0xc024008cb8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958740)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024009018 sp=0xc024008e68 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958730)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc0240091c8 sp=0xc024009018 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958720)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024009378 sp=0xc0240091c8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958710)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024009528 sp=0xc024009378 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958700)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc0240096d8 sp=0xc024009528 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029586f0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024009888 sp=0xc0240096d8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029586e0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024009a38 sp=0xc024009888 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029586d0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024009be8 sp=0xc024009a38 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029586c0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc024009d98 sp=0xc024009be8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc0029586b0)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc024009f48 sp=0xc024009d98 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc0029586a0)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc02400a0f8 sp=0xc024009f48 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958690)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc02400a2a8 sp=0xc02400a0f8 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958680)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc02400a458 sp=0xc02400a2a8 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958670)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc02400a608 sp=0xc02400a458 pc=0xbab024
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x35251e0, 0xc0015db7d0, 0x1, 0xc002958660)
        /home/circleci/project/compiler/interface.go:117 +0x760 fp=0xc02400a7b8 sp=0xc02400a608 pc=0xbab4f0
github.com/tinygo-org/tinygo/compiler.getTypeCodeName(0x3525260, 0xc0015db860, 0x1, 0xc002958650)
        /home/circleci/project/compiler/interface.go:133 +0x294 fp=0xc02400a968 sp=0xc02400a7b8 pc=0xbab024

goroutine 5 [syscall]:
os/signal.signal_recv(0x0)
        /usr/local/go/src/runtime/sigqueue.go:139 +0x9c
os/signal.loop()
        /usr/local/go/src/os/signal/signal_unix.go:23 +0x22
created by os/signal.init.0
        /usr/local/go/src/os/signal/signal_unix.go:29 +0x41
```

**Comment by aykevl** [comment]
This bug has been fixed in the dev branch: https://github.com/tinygo-org/tinygo/pull/474
Duplicate issue: #309.

We will probably have a release in the near future which includes this fix.

**Comment by 88250** [comment]
> This bug has been fixed in the dev branch: #474
> Duplicate issue: #309.
> 
> We will probably have a release in the near future which includes this fix.

Thanks, it is very exciting!
